ansible/roles/debops.prosody/defaults/main.yml

```diff
@@ -0,0 +1,162 @@
+#
+# Default PKI realm to use.
+prosody__pki_realm: '{{ ansible_local.pki.realm
+                       if (ansible_local|d() and ansible_local.pki|d() and
+                           ansible_local.pki.realm|d())
+                       else "domain" }}'
+                                                                   # ]]]
+# .. envvar:: prosody__pki_crt_filename [[[
+#
+# Default CRT file name to use.
+prosody__pki_crt_filename: '{{ ansible_local.pki.crt
+                              if (ansible_local|d() and ansible_local.pki|d() and
+                                  ansible_local.pki.crt|d())
+                              else "default.crt" }}'
+                                                                   # ]]]
+# .. envvar:: prosody__pki_key_filename [[[
+#
+# Default private key file name to use.
+prosody__pki_key_filename: '{{ ansible_local.pki.key
+                              if (ansible_local|d() and ansible_local.pki|d() and
+                                  ansible_local.pki.key|d())
+                              else "default.key" }}'
+                                                                   # ]]]
+# .. envvar:: prosody_pki_hook_name [[[
+#
+# Name of the hook script which will be stored in hook directory.
+prosody_pki_hook_name: 'prosody'
+
+                                                                   # ]]]
+# .. envvar:: prosody_pki_hook_path [[[
+#
+# Directory with PKI hooks.
+prosody_pki_hook_path: '{{ ansible_local.pki.hooks
+                         if (ansible_local|d() and ansible_local.pki|d() and
+                             ansible_local.pki.hooks|d())
+                         else "/etc/pki/hooks" }}'
+
+                                                                   # ]]]
+# .. envvar:: prosody_pki_hook_action [[[
+#
+# Specify how changes in PKI should affect prosody, either 'reload' or 'restart'.
+prosody_pki_hook_action: 'reload'
+
+                                                                   # ]]]
+                                                                   # ]]]
+# .. envvar:: prosody__admins [[[
+#
+# list of admins jabber accounts of this prosody instance
+prosody__admins: []
+                                                                   # ]]]
+# .. envvar:: prosody__config_global: [[[
+#
+# main prosody configuration
+# @see prosody.im
+prosody__config_global:
+  admins: '{{ prosody__admins }}'
+  modules_enabled:
+    - "roster"
+    - "saslauth"
+    - "tls"
+    - "dialback"
+    - "disco"
+    - "private"
+    - "vcard"
+    - "privacy" #@TODO check version < 0.10
+    - "compression" #@TODO check lua-zlib
+    - "version"
+    - "uptime"
+    - "time"
+    - "ping"
+    - "pep"
+    - "admin_adhoc"
+    - "posix"
+    - "groups"
+    - "carbon"
+    - "mam"
+    - "blocking"
+    - "smacks"
+    #- "s2s_auth_dane"
+  allow_registration: False
+  daemonize: True
+  pidfile: "/var/run/prosody/prosody.pid"
+  #ssl:
+  #  key: '..'
+  #  certificate: '..'
```

Comment:
This should be hooked up by default to the DebOps PKI using the `prosody__pki_*` variables defined earlier.